src/templateAccessibilityElementsContentRule.ts

```diff
@@ -32,21 +32,22 @@ class TemplateAccessibilityAnchorContentVisitor extends BasicTemplateAstVisitor
 
 export class Rule extends Rules.AbstractRule {
   static readonly metadata: IRuleMetadata = {
-    description: 'Ensures that the anchor element has some content in it',
+    description: 'Ensures that the heading and anchor elements have content in it',
```

Comment:
In the same way, I see that button have a [similar need](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role#Accessibility_concerns).
Can we again broaden the rule ?

Reply:
Done